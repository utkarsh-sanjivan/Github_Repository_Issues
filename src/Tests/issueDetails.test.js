import React from 'react';
import { create } from "react-test-renderer";
import { CheckOutlined } from '@ant-design/icons';
import StatusTag from './../component/StatusTag/index';
import MarkdownGithub from 'react-markdown-github';

test('Test Status Tag Component in Issue Details', () => {
  const component = create(
    <StatusTag
        color='#fff'
        icon={<CheckOutlined/>}
        text={'Test Text'}
    />
);
  expect(component.toJSON()).toMatchSnapshot()
});

test('Test Github Markdown Component in Issue Details', () => {
  const component = create(
    <MarkdownGithub
        source='This is a test github comment body'
    />
);
  expect(component.toJSON()).toMatchSnapshot()
});