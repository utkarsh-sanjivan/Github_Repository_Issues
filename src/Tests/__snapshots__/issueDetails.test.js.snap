// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Github Markdown Component in Issue Details 1`] = `
<p>
  This is a test github comment body
</p>
`;
